src/olympia/reviewers/templates/reviewers/includes/history.html

```diff
@@ -9,6 +9,9 @@
     {% if record.details %}
       {# <pre> rather than <div> so that white space is preserved on copy #}
       <pre class="light history-comment">{{ record.details.comments }}</pre>
+        {% if record.details.reason %}
+          <pre class="light history-comment">Reason: {{ record.details.reason }}</pre>
```

Comment:
Can you attach a screenshot to the PR - I'm interested how to this actually looks (does it just get concatenated together with a space when rendered; line breaks are okay;etc).

Reply:
There is a screenshot in the issue. Is that sufficient?

Reply:
Hah, you filed the issue _after_ you'd completed the patch.  Sure - I didn't look there.